eng/workloads/workloads.csproj

```diff
@@ -10,8 +10,6 @@
     <VSTemp>$(WorkloadIntermediateOutputPath)VS/</VSTemp>
     <WorkloadOutputPath>$(ArtifactsBinDir)workloads/</WorkloadOutputPath>
     <WorkloadOutputPath Condition="'$(workloadArtifactsPath)' != ''">$(workloadArtifactsPath)/</WorkloadOutputPath>
-    <PackageSource>$(ArtifactsShippingPackagesDir)</PackageSource>
-    <PackageSource Condition="'$(workloadPackagesPath)' != ''">$(workloadPackagesPath)/</PackageSource>
```

Comment:
Does this mean that this cannot get overriden any longer? cc: @danegsta in case he is aware if we depend on this.

Reply:
Where does `$(workloadPackagesPath)` get set? I didn't find any reference to it in the repo.

Reply:
Should be fine as long as the workload still builds correctly